protos/lightstreamer.proto

```diff
@@ -10,7 +10,7 @@ message ServerInfo {
  string nodeVersion = 4;
  string nodeStatus = 5;
  uint64 blockHeight = 6;
- bytes blockHash = 7;
+ string blockHash = 7;
```

Comment:
I was looking at the types from the SDK, and noticed this is a string. We had it as bytes before. Which one is right?

Reply:
Whatever the sdk returns is likely what you want. It could be represented as either, as the returned string is hex encoded bytes.